frontend/src/reducers/plannerSlice.ts

```diff
@@ -326,16 +326,20 @@ const plannerSlice = createSlice({
       // For now, keep courses hidden if summer column is also hidden. If we want
       // to remove the courses if summer term is toggled off, then 'term complete'
       // must also be reset.
-      // if (!state.isSummerEnabled) {
-      //   for (let i = 0; i < state.numYears; i++) {
-      //     const courses = state.years[i].T0;
-      //     courses.forEach((course) => {
-      //       state.courses[course].plannedFor = null;
-      //       state.unplanned.push(course);
-      //     });
-      //     state.years[i].T0 = [];
-      //   }
-      // }
+      if (!state.isSummerEnabled) {
+        for (let i = 0; i < state.numYears; i++) {
+          const courses = state.years[i].T0;
+          courses.forEach((course) => {
+            if (state.courses[course].isMultiterm) {
```

Comment:
I'm pretty sure you can just call the `unschedule` reducer and chain it together to remove the course in the summer column. Rather than having it as seperate cases (i.e. both non multiterm and multiterm courses call `plannerSlice.caseReducers.unschedule`?